nodejs/week1/avg.js

```diff
@@ -0,0 +1,12 @@
+const numbers = process.argv
+  .slice(2)
+  .map(Number)
```

Comment:
I like your solution for this problem. Maybe I would suggest separating the logic a bit, for example, first, get the numbers with `process.argv` and then in another variable do the mapping and filtering. It will help you make your code more readable to others and help you detect any errors.